src/preferences.js

```diff
@@ -71,83 +77,104 @@ i18n.on('loaded', () =>
     translatePage(i18n.language);
 });
 
-$(() =>
+function refreshContent()
+{
+    usersStyles = getUserPreferences();
+}
+
+function updateUserPreferences()
+{
+    ipcRenderer.send('PREFERENCE_SAVE_DATA_NEEDED', preferences);
+}
+
+function changeValue(type, new_val)
+{
+    preferences[type] = new_val;
```

Comment:
We try to avoid the snake case for variables names. This would be better:

```suggestion
function changeValue(type, newVal)
{
    preferences[type] = newVal;
```